packages/composite-checkout/src/components/credit-card-fields.js

```diff
@@ -0,0 +1,151 @@
+				fill="black"
+			/>
+			<rect x="44.7258" y="18.9998" width="17.4205" height="13.3329" stroke="#C9356E" />
+		</svg>
+	);
+}
+
+export function LockIcon( { className } ) {
```

Comment:
Do you expect that `LockIcon` or `CVV` will be used outside this file? If not, I'd suggest we don't export them. That way it reduces the public API of this file and becomes an implementation detail that we can change at our leisure. If we do export them, then any time we want to change them we have to search the rest of the code base to see where else they might be used.

Reply:
Let's not export them until we need them outside this component (at which point we can extract them). I'll update it so that they're not exportable.